src/daemon/plugin.h

```diff
@@ -34,8 +34,14 @@
 #include "meta_data.h"
 #include "utils_time.h"
 
+#include <inttypes.h>
 #include <pthread.h>
 
+#ifdef WIN32
+// ERROR is previously defined in wingdi.h, included from windows.h.
```

Comment:
I do recall from a previous life that there was something like -DWIN32_LEAN_AND_MEAN which reduced the number of windows headers included. Would that help?